PinwheelSDK/Classes/Pinwheel.swift

```diff
@@ -9,12 +9,13 @@ import Foundation
 
 import UIKit
 import WebKit
-import SwiftUI
 
 public protocol PinwheelDelegate {
-    func onEvent(_ event: PinwheelActionEvent)
-    func onExit(_ event: PinwheelExitEvent)
-    func onSuccess(_ event: PinwheelSuccessEvent)
+    func onEvent(name: PinwheelEventType, event: PinwheelEventPayload?)
+    func onExit(_ event: PinwheelError?)
```

Comment:
Not sure if it matters but the pattern we're following is that this is named error instead of event

Reply:
This would be consistent with what you did below for onError too